src/network.c

```diff
@@ -3059,6 +3026,32 @@ static int network_config(oconfig_item_t *ci) /* {{{ */
   return 0;
 } /* }}} int network_config */
 
+static bool check_send_notification(const notification_t *n) {
```

Comment:
Please keep the changed code in same place. Moving the code around makes diffs larger & reviewing the actual changes harder.

Reply:
Ok, done.
Thank you.

Reply:
Thanks!